.circleci/config.yml

```
@@ -0,0 +1,78 @@
+version: 2.1
+
+executors:
+  dst-go-executor:
+    docker:
+      - image: cimg/go:1.14.1
+    environment:
+      RESULTS: /tmp/results
+      ARTIFACTS: /tmp/artifacts
+      TERM: xterm
+    working_directory: ~/dst-go
+
+commands:
+  checkout-code:
+    steps:
+      - checkout
+      - run:
+          name: Check if go.mod requires update
+          command: go list -mod=readonly ./...
+
+  make-results-artifacts-dir:
+    steps:
+      - run: mkdir -p $RESULTS && ls -l /tmp
+      - run: mkdir -p $ARTIFACTS && ls -l /tmp
+
+  upload-results-artifacts:
+    steps:
+      - store_test_results:
+          path: /tmp/results
+      - store_artifacts:
+          path: /tmp/artifacts
+
+  install-txt-to-html-coverter-aha:
+    steps:
+      - run: sudo apt-get update
+      - run: sudo apt-get install aha
+
+jobs:
+
+  lint:
+    executor: dst-go-executor
+    steps:
+      - checkout-code
+      - make-results-artifacts-dir
+      - run:
+          name: Install linter
+          command: curl -sfL https://raw.githubusercontent.com/golangci/golangci-lint/master/install.sh| sh -s -- -b $(go env GOPATH)/bin v1.28.2
```

```suggestion
          command: curl -sfL https://raw.githubusercontent.com/golangci/golangci-lint/master/install.sh | sh -s -- -b $(go env GOPATH)/bin v1.28.2
```
nitpick... consistent formatting

---

Updated in e2db49f07a920b374190486bb35716ee39bc71b4.